src/frontend/src/main.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.scss";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import Dashboard from "./pages/Dashboard/Dashboard";
+import Project from "./pages/Project/Project";
+import Group from "./pages/Group/Group";
+import Layout from "./pages/Layout";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: (
+      <Layout>
+        <Dashboard />
+      </Layout>
+    ),
+  },
+  {
+    path: "/project",
+    element: (
+      <Layout>
+        <Project />
+      </Layout>
+    ),
+  },
+  {
+    path: "/group",
```

Comment:
Having gone down the road of magic path strings over the app, let's create a config/ module with a routeConfig dictionary for all of these (we can re-use it in AppHeader too)